build-tools/scripts/DotNet.targets

```diff
@@ -4,14 +4,14 @@
     <_BinlogDateTime>$([System.DateTime]::Now.ToString("yyyyMMddTHHmmss"))</_BinlogDateTime>
   </PropertyGroup>
   <Target Name="BuildExternal">
-    <MSBuild
-        Projects="$(_Root)\external\monodroid\monodroid.proj"
-        Properties="DebuggingToolsOutputDirectory=$(MicrosoftAndroidSdkOutDir);CompatTargetsOutputDirectory=$(XAInstallPrefix)xbuild\Novell"
+    <Exec
+        Command="&quot;$(DotNetPreviewTool)&quot; build monodroid.proj -c $(Configuration) -p:DebuggingToolsOutputDirectory=$(MicrosoftAndroidSdkOutDir) -p:CompatTargetsOutputDirectory=$(XAInstallPrefix)xbuild/Novell -v:n"
+        WorkingDirectory="$(_Root)external\monodroid"
     />
   </Target>
   <Target Name="PrepareJavaInterop">
     <Exec
-        Command="&quot;$(DotNetPreviewTool)&quot; build -t:Prepare Java.Interop.sln -c $(Configuration) -p:JdksRoot=$(JavaSdkDirectory) -p:DotnetToolPath=$(DotNetPreviewTool) -bl:$(_Root)bin/Build$(Configuration)/msbuild-$(_BinlogDateTime)-prepare-java-interop.binlog"
+        Command="&quot;$(DotNetPreviewTool)&quot; build -t:Prepare Java.Interop.sln -c $(Configuration) -p:JdksRoot=$(JavaSdkDirectory) -p:DotnetToolPath=$(DotNetPreviewTool) -v:n"
         WorkingDirectory="$(_Root)external\Java.Interop"
     />
```

Comment:
It seems like we lose the `.binlog` on these commands now. I don't see how the `make` target will create these, as `<Exec/>` runs `dotnet build` in a new process?

Reply:
Oh right, we would only get the normal output from exec in the binlog created by the make file. Let me swap this to use the default minimal output verbosity but also produce binlogs again.